README.md

```diff
@@ -7,6 +7,43 @@
 
 Service Utilization API for tracking resource usage and metrics reporting
 
+## Client
+
+The module exposes a client, named UtapiClient. Projects can use this client to
+push metrics directly to the underlying datastore(Redis) without the need of an
+extra HTTP request to Utapi.
+
+```js
+import { UtapiClient } from 'utapi';
+
+const config = {
+    redis: {
+        host: '127.0.0.1',
+        port: 6379
+    }
+}
+const c = new UtapiClient(config);
+
+c.pushMetric('createBucket', '3d534b1511e5630e68f0', { bucket: 'demo' });
```

Comment:
Only comment is you can add a comment to explain the parameters if you don't think the people reading the README.md will be familiar with Utapi already. Mostly it is the reqUid that is not clear at first glance what it represents.